test/github-auth.js

```diff
@@ -68,6 +68,15 @@ class FakeGithub {
       'integrations.getInstallationRepositories': async() => {
         return this.repositories;
       },
+      'users.getForUser': async ({username}) => {
+        let user = _.find(this.github_users, {username});
+        if (user) {
+          user.id = parseInt(user.id, 10);
```

Comment:
Well, in real github payloads id is always a number. users.getForId uses id in form of a string - I have no idea why :) That's, I suspect, where toString comes from? ;) But for users.getForUser I need a number.